### PR TITLE
DEV: update modules-api-ref.md for 7.4-rc2

### PR DESCRIPTION
**Pull request by dwdougherty**
Update to modules-api-ref.md from redis/redis source code.

**Note to reviewers**: This document is generated by software, so any language anomalies or incorrect links will need to be fixed in the redis/redis source code, not here.

### CODE REVIEW

File: content/develop/reference/modules/modules-api-ref.md [hunk 1]
```diff
@@ -1934,9 +1932,8 @@ Redis will accumulate all the calls to this function in the context of
 the callback, and will propagate all the commands wrapped in a MULTI/EXEC
 transaction. However when calling this function from a threaded safe context
 that can live an undefined amount of time, and can be locked/unlocked in
```
Comment (andy-stark-redis) [inline content/develop/reference/modules/modules-api-ref.md:1934]: Guessing the "in" shouldn't be there?
```suggestion
that can live an undefined amount of time, and can be locked/unlocked
```

File: content/develop/reference/modules/modules-api-ref.md [hunk 2]
```diff
@@ -3840,7 +3840,7 @@ AOF rewrite, and so forth). In this section we define this API.
 
 Register a new data type exported by the module. The parameters are the
 following. Please for in depth documentation check the modules API
-documentation, especially [/develop/reference/modules/modules-native-types]({{< relref "/develop/reference/modules/modules-native-types" >}}).
+documentation, especially [https://redis.io/topics/modules-native-types](https://redis.io/topics/modules-native-types).
```
Comment (andy-stark-redis) [inline content/develop/reference/modules/modules-api-ref.md:3843]: The link seems to go to the docs homepage, so presumably dodgy.

File: content/develop/reference/modules/modules-api-ref.md [hunk 3]
```diff
@@ -4489,7 +4489,7 @@ latency-monitor-threshold.
 ## Blocking clients from modules
 
 For a guide about blocking commands in modules, see
-[/develop/reference/modules/modules-blocking-ops]({{< relref "/develop/reference/modules/modules-blocking-ops" >}}).
+[https://redis.io/topics/modules-blocking-ops](https://redis.io/topics/modules-blocking-ops).
```
Comment (andy-stark-redis) [inline content/develop/reference/modules/modules-api-ref.md:4492]: Dodgy link again.

**Review by andy-stark-redis — CHANGES_REQUESTED** [review]
Seems to have a couple of dodgy links but basically OK.

File: content/develop/reference/modules/modules-api-ref.md (hunk 1 above)
Comment (dwdougherty) [reply content/develop/reference/modules/modules-api-ref.md:1934]: There's no point in changing text, as it's all generated from the redis/redis source code. That said, I'll make a note of this for when I start spelunking and requesting changes.

File: content/develop/reference/modules/modules-api-ref.md (hunk 2 above)
Comment (dwdougherty) [reply content/develop/reference/modules/modules-api-ref.md]: Same issue: these links are embedded in source code, so I'll need to hunt them down and fix them there.

Fixed with next push.

File: content/develop/reference/modules/modules-api-ref.md (hunk 3 above)
Comment (dwdougherty) [reply content/develop/reference/modules/modules-api-ref.md]: Same issue: these links are embedded in source code, so I'll need to hunt them down and fix them there.

Fixed with next push.

**Pull request merged**